examples/player/simid_player.js

```diff
@@ -198,6 +207,8 @@ class SimidPlayer {
         this.onRequestChangeAdDuration.bind(this));
     this.simidProtocol.addListener(CreativeMessage.GET_MEDIA_STATE, this.onGetMediaState.bind(this));
     this.simidProtocol.addListener(CreativeMessage.LOG, this.onReceiveCreativeLog.bind(this));
+    this.simidProtocol.addListener(CreativeMessage.REQUEST_EXPAND, this.onExpandResize.bind(this));
+    this.simidProtocol.addListener(CreativeMessage.REQUEST_COLLAPSE, this.displayNonLinearCreative.bind(this));
```

Comment:
took a little looking to figure this out. But i think you are ending up with some weird code (probably buggy) because you are trying to use the same function for setting up the non linear are you are for a request collapse call.  Better instead to just make a request collapse function.